bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -14,19 +14,47 @@
 #pragma once
 #include <optional>
 #include "IStateTransfer.hpp"
+#include "ReservedPages.hpp"
+#include "Serializable.h"
 
 namespace bftEngine {
 
-class ControlStateManager {
+class ControlStateManager : public ResPagesClient<ControlStateManager, 1, 1> {
  public:
   void setStopAtNextCheckpoint();
-  std::optional<uint64_t> getStopCheckpointToStopAt();
+  std::optional<bool> getStopCheckpointToStopAt();
```

Comment:
Why not return the actual checkpoint?

Reply:
Nit: Naming. `getCheckpointToStopAt` is more legible.

Reply:
fixed